example/ida/utils.py

```diff
@@ -126,20 +134,23 @@ def from_ExprOp(self, expr):
             return (' ' + expr._op + ' ').join([self.str_protected_child(arg, expr)
                                                 for arg in expr._args])
         return (expr._op + '(' +
-                ', '.join([self.from_expr(arg) for arg in expr._args]) + ')')
+                ', '.join([
```

Comment:
Given these lines are changed, `[]` can be removed

Reply:
done